src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const transformedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .reduce((acc, element) => {
+      const splitedEl = element.split(':').map((el) => el.trim());
+
+      acc[splitedEl[0]] = splitedEl[1];
```

Comment:
Ensure that `splitedEl[0]` and `splitedEl[1]` are valid. If the input string is malformed, this could lead to undefined values. Consider adding a check to ensure both elements exist before assigning them to the object.